openapi/api.yml

```diff
@@ -224,7 +224,6 @@ components:
       type: object
       required:
         - roles
-        - azure_ad_subject_id
```

Comment:
Know it is not directly related to this PR, but shouldn't `email_address` be required?

Reply:
Surely we need the email to lookup the user in the DB? Unless I'm missing something? 

Does it not use the email to lookup the account ID?

Reply:
We're not looking up by email.

We are searching by `account_id` which is a path parameter; and if provided `azure_ad_subject_id`.
If `email` is provided we update it. Nothing else.